var/spack/repos/builtin/packages/superlu/package.py

```diff
@@ -43,7 +43,10 @@ class Superlu(CMakePackage, Package):
     )
 
     build_system(
-        conditional("cmake", when="@5:"), conditional("generic", when="@:4"), default="cmake"
+        conditional("cmake", when="@5:"),
+        conditional("generic", when="@:4"),
+        conditional("cmake", when="platform=windows"),
```

Comment:
Do you know if cmake builds were available for versions before 5? I would guess they weren't in which case this should be replaced with a separate `conflicts(platform=windows, when=@:4)`.